profiles/maryland.pjs

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "maryland",
+    "type": "oai_verbs",
+    "metadata_prefix": "oai_qdc",
+    "endpoint_url": "http://webconfig.digitalmaryland.org/oai/oai.php",
+    "contributor": {
+        "@id": "http://dp.la/api/contributor/digital-maryland",
```

Comment:
There is a discrepancy between the contributor and provider `@id` property value. 

`http://dp.la/api/contributor/maryland` vs `http://dp.la/api/contributor/digital-maryland` Not sure which one is correct.